### DISCUSSION TITLE
Pull data from title into attribute

### DISCUSSION THREAD

**Discussion by Beefthong** [datafeedr/wordpress-plugins#35] [Ideas]
Not sure if this is a request or plead for help. I'm running a bed website with hundreds of products, most identical but different sizes. The specific sizes are in the titles across merchants - it seems to be rather common. But the attribute is not common.

Is it possible to fetch the number from the title and import as an attribute? For instance a bed called Hyoer Fancy Continental 140x200, would be 140x200 size attribute.

**Comment by EricBusch** [comment]
Hi

Yes, it's possible but will require some programmatic parsing of the title to extract that data.

You might need to write a regular expression to extract the size from the title. Then, once you have the size extracted from the title, you can use code like this to import the size as an attribute: https://datafeedrapi.helpscoutdocs.com/article/51-add-size-as-a-product-attribute

Hope this helps!

Eric

**Reply by Beefthong** [reply]
It sure does, thank you :)
